lib/livebook_web/live/session_live/app_teams_live.ex

```diff
@@ -180,15 +180,14 @@ defmodule LivebookWeb.SessionLive.AppTeamsLive do
     ~H"""
     <div class="flex flex-col gap-6">
       <p class="text-gray-700">
-        Deploy this app to your cloud infrastructure. The app will be pushed to the Livebook
-        Teams app servers associated with the deployment group that you choose.
+        Deploy this app to your cloud infrastructure using the <.workspace hub={@hub} /> workspace.
       </p>
 
       <%= if @deployment_group do %>
         <div class="flex flex-col gap-2">
           <div class="flex justify-between items-center">
             <p class="text-gray-700">
-              Using deployment group in <.workspace hub={@hub} /> workspace:
+              Last deployed to:
```

Comment:
If the user goes through the initial flow, creates the deployment group, then closes the modal and opens it later, we will show "Last deployed to" even though there's no deployment.

What about "Deploying to"?